omnibus/config/software/datadog-agent.rb

```diff
@@ -136,7 +136,7 @@
 
   if not bundled_agents.include? "trace-agent"
     platform = windows_arch_i386? ? "x86" : "x64"
-    command "invoke trace-agent.build --install-path=#{install_dir} --major-version #{major_version_arg} --flavor #{flavor_arg}", :env => env
+    command "dda inv -- trace-agent.build --install-path=#{install_dir} --major-version #{major_version_arg} --flavor #{flavor_arg}", :env => env
```

Comment:
can we also add `-e` to be consistent and get more data?

Reply:
Added!